modules/zookeeper/src/test/java/org/apache/ignite/spi/discovery/zk/ZookeeperDiscoverySpiTestUtil.java

```diff
@@ -18,29 +18,44 @@
 
 import java.io.File;
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
+
 import org.apache.curator.test.InstanceSpec;
-import org.apache.ignite.spi.discovery.tcp.ipfinder.zk.curator.TestingCluster;
+import org.apache.curator.test.TestingCluster;
 import org.apache.ignite.IgniteException;
+import org.jetbrains.annotations.Nullable;
 
 /**
  * Utility to run regular Ignite tests with {@link org.apache.ignite.spi.discovery.zk.ZookeeperDiscoverySpi}.
  */
 public class ZookeeperDiscoverySpiTestUtil {
+    /** Property name for Zookeeper's election port bind retry attempts count. */
+    public static final String ZK_ELECTION_PORT_BIND_RETRY = "electionPortBindRetry";
+
+    /** Property name for Zookeeper's 'enable admin server' flag. */
+    public static final String ZK_ENABLE_ADMIN_SERVER = "admin.enableServer";
+
     /**
      * @param instances Number of instances in cluster.
+     * @param customProps Custom configuration properties for every server.
      * @return Test cluster.
      */
-    public static TestingCluster createTestingCluster(int instances) {
-        return createTestingCluster(instances, 0);
+    public static TestingCluster createTestingCluster(int instances, @Nullable Map<String,Object>[] customProps) {
```

Comment:
Let's introduce 
```
public static TestingCluster createTestingCluster(int instances) {
    return createTestingCluster(instances, 0, null);
}
```

It will reduce the number of changes in other places, I think.

Reply:
fixed